content/posts/helpful-webassembly-resources.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Helpful WebAssembly Resources - A List for Kubernetes and WebAssembly"
+description: "A useful set of lists for using WebAssembly in Kubernetes instead of containers."
+date: 2022-12-05 00:00:00 +0000 UTC
+authorname: "Ralph Squillace"
+author: "@ralph_squillace"
+authorlink: "https://twitter.com/ralph_squillace"
+tags: ["kubernetes", "containerd","spiderlightning", "WebAssembly", "spin", "Wasm", "WASI"]
+---
+
+Things are moving so fast that often I get asked to host a quick list of the recent things people can read or try for WebAssembly (wasm) outside of the browser, specifically about the WebAssembly System Interface (WASI) and WebAssembly components. Here's some of the best links to various parts of things that did not work so well and those that do. I am leaving out **some** links that we're not pursuing, but at the bottom in the appendix I'll drop a few of those.
+
+In particular, while Custom Resource Descriptions (CRDs) are a great way to integrate things into Kubernetes, our objectives are to simplify using Kubernetes and not extend it so that the model you use is more and not less complex. As a result, these links focus on the "heritage" of modeling host processes as "pods" in kubernetes just like containers do.
```

Comment:
```suggestion
In particular, while Custom Resource Descriptions (CRDs) are a great way to integrate things into Kubernetes, our objectives are to simplify using Kubernetes and not extend it so that the model you use is less complex. As a result, these links focus on the "heritage" of modeling host processes as "pods" in kubernetes just like containers do.
```

Reply:
yup